02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html

```diff
@@ -70,7 +68,7 @@
         return new Weather()
         {
             // Make sure we only get this data AFTER trading day - don't want forward bias.
-            Time = DateTime.Parse(data[0]).AddHours(20),
+            Time = DateTime.ParseExact(data[0], "yyyyMMdd", null).AddHours(20),
```

Comment:
In the Python version, we have:
```python
weather.Time = datetime.strptime(data[0], '%Y-%m-%d') + timedelta(hours=20)
```
should it not be changed too?

Reply:
Aaah right. Thanks for catching that. Should be 
`datetime.strptime(data[0], '%Y%m%d') + timedelta(hours=20)`